src/term/net_to_term.rs

```diff
@@ -503,6 +582,7 @@ impl Term {
         snd.fix_names(id_counter, book);
       }
       Term::Let { .. } => unreachable!(),
+      Term::Tup { .. } => todo!(),
```

Comment:
This needs to be done, should be identical to Sup one. I don't understand how it's not crashing by leaving this as a todo!(). If it's actualyl unreadcheable then put that instead. Also, maybe have a look if this is disabled for some reason?